### PR TITLE
fix: add padding to bottom of global mode list

### LINKED ISSUE #21724: Add padding to bottom of global mode screen so UI is not flush against bottom
The list of projects is completely flush with the bottom of the app. Add some padding to get some breathing room here. It feels strange when using. 

[Gif of behavior](http://g.recordit.co/IU3dh3W5Wf.gif)

Screen Shot 2022-05-31 at 12 06 14 PM

┆Issue is synchronized with this [Jira Task](https://cypress-io.atlassian.net/browse/UNIFY-1095) by [Unito](https://www.unito.io)
┆author: Jennifer Shehane
┆epic: UI/UX Improvements
┆friendlyId: UNIFY-1095
┆priority: Lowest
┆sprint: Unification Release Backlog
┆taskType: Task

### PR DESCRIPTION
**Pull request by viniciuspietscher**
- Closes #21724

This PR was Co-authored-by: Mark Noonan <mark@cypress.io>

### User facing changelog

Add 24px of padding to bottom of global mode list.

### Additional details

- Why was this change necessary?
 Projects list on GlobalPage were flush with the end of the page.

- What is affected by this change?
 GlobalPage UI

### Steps to test

Add a small viewport with `cy.viewport()` on packages/launchpad/src/global/GlobalPage.cy.tsx

### How has the user experience changed?

Before:
before

After:
after

### PR Tasks

- [x] Have tests been added/updated?
- [ ] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

File: packages/launchpad/src/global/GlobalPage.cy.tsx [hunk 1]
```diff
@@ -68,6 +68,13 @@ describe('<GlobalPage />', { viewportHeight: 900, viewportWidth: 1200 }, () => {
     it('renders projects', () => {
       cy.findByText(testProject).should('be.visible')
       cy.findByText(testProjectPath).should('be.visible')
+      // validate that there is some padding on the bottom
+      cy.get('[data-cy="project-card"]')
+      .parent()
+      .invoke('outerHeight')
+      .then((parentHeight) => {
+        cy.get('[data-cy="project-card"]').invoke('outerHeight').should('be.lt', parentHeight)
```
Comment (marktnoonan) [inline packages/launchpad/src/global/GlobalPage.cy.tsx:76]: Looks like we don't have any Percy snapshots for this global mode project list, let's add one here.

```suggestion
        cy.get('[data-cy="project-card"]').invoke('outerHeight').should('be.lt', parentHeight)
        cy.percySnapshot()
```

**Review by lmiller1990 — APPROVED** [review]

**Review by tbiethman — COMMENTED** [review]
Fix looks good and addresses the immediate issue :+1:

One thought I had is that we could add this padding globally within the launchpad, like we do for the horizontal padding: https://github.com/cypress-io/cypress/blob/b0e32f8f8ea196438e12308b0ab36fde1c17bd02/packages/launchpad/src/Main.vue#L15

This would ensure that all components rendered within Main (including the project list) would have some level of bottom padding. But it would also add more padding to components that already have their own, so we'd have to double check for those. 

@marktnoonan thoughts on this approach? If we want to circle back around with another issue to make the padding consistent, I'd be fine with that too. The project list is the most egregious at this point, but the browser selection and error views are also flush with the window when scrolled:

Screen Shot 2022-06-06 at 11 44 14 AM

Screen Shot 2022-06-06 at 11 45 14 AM

**Comment by viniciuspietscher** [comment]
> That's a good suggestion @tbiethman. @viniciuspietscher why don't you go ahead and make that change in Main.vue. Let's drop the component test changes from this PR and let Percy detect if the padding at the bottom of global mode changes.

@marktnoonan updated the PR with the suggested changes. :+1:

**Review by ryanjwilke — APPROVED** [review]
This does look like an improvement from what we have today.

@marktnoonan I would say that we should have a look together at the padding and scrollbars together as I've seen a lot of things that don't quite match up to our expectations and it's not easy to show some of these things in Figma either.

**Review by tbiethman — APPROVED** [review]

**Comment by lmiller1990** [comment]
PRs from external contributors consistently exhibit failures on these 7 tests [link to CI](https://app.circleci.com/pipelines/github/cypress-io/cypress/39758/workflows/2dbb244e-23e6-4ba3-be1b-d0b8c033d542/jobs/1628514). I don't know why, but it has happened multiple times recently, and I'd like to find out why, it's blocking lots of great contributions. Posting this here to have something to refer to while debugging this issue.

image

**Pull request merged**